mantis-runtime-executor/src/main/java/io/mantisrx/server/worker/jobmaster/WorkerMetricHandler.java

```diff
@@ -19,6 +19,7 @@
 import static io.mantisrx.server.core.stats.MetricStringConstants.*;
 
 import io.mantisrx.runtime.descriptor.StageScalingPolicy;
+import io.mantisrx.runtime.descriptor.StageScalingPolicy.ScalingReason;
```

Comment:
Remove this and keep the code consistent.

Reply:
Sure. It looks like it was automatically imported by Intellij.